lib/bamboo/test.ex

```diff
@@ -203,6 +203,26 @@ defmodule Bamboo.Test do
     end
   end
 
+  defmacro refute_email_delivered_with(email_params) do
```

Comment:
Thanks for this!! Thoughts on adding some documentation for it?

Reply:
249ce54